spring-data-jdbc-plus-sql/src/main/java/com/navercorp/spring/data/jdbc/plus/sql/convert/AggregateResultJdbcConverter.java

```diff
@@ -236,19 +249,18 @@ private List<Map<String, Object>> extractData(
 
 		Map<Object, ExtractedRow> extractedRows = new LinkedHashMap<>();
 
-		while (!resultSetHolder.isDone() && resultSetHolder.next()) {
-			Map<String, Object> entityMap = this.mapSingleTableRow(
-				persistentEntity, resultSetHolder.getResultSet());
+		while (resultSet.next()) {
```

Comment:
ref. [spring-jdbc RowMapperResultSetExtractor](https://github.com/spring-projects/spring-framework/blob/810da11bb542c681036a3a8fb4585eae7492c4e1/spring-jdbc/src/main/java/org/springframework/jdbc/core/RowMapperResultSetExtractor.java#L93-L95)
This extract makes `ResultSet` to `List<T>`.

We've managed the AggregateResult results in this method, I bring the iteration logic here.